test/integration/lib/foxbox_process_manager.js

```diff
@@ -35,6 +35,12 @@ var foxbox_process_manager = (function() {
       'User DB not found!');
     _removeFileIfItExists('/.local/share/foxbox/webpush.sqlite',
       'webpush DB not found!');
+    _removeFileIfItExists('/.local/share/foxbox/foxbox.conf',
+      'Foxbox config file not found!');
+    _removeFileIfItExists('/.local/share/foxbox/taxonomy_tags.sqlite',
```

Comment:
Nit: Rename the function => We don't only delete databases anymore. I wonder if we shouldn't delete every file under `.local/share` except the certificates. Here's a way to do it https://github.com/fxbox/foxbox/blob/master/test/selenium/lib/foxbox_process_manager.js#L44